src/applications/personalization/profile360/util/transactions.js

```diff
@@ -36,6 +36,26 @@ const GENERIC_ERROR_CODES = new Set([
   'VET360_CORE502'
 ]);
 
+const MVI_NOT_FOUND_ERROR_CODES = new Set([
+  'VET360_MVI201'
+]);
+
+const MVI_ERROR_CODES = new Set([
+  'VET360_MVI100',
+  'VET360_MVI101',
+  'VET360_MVI200',
+  'VET360_MVI202',
+  'VET360_MVI203'
+]);
+
+const LOW_CONFIDENCE_ADDRESS_ERROR_CODES = new Set([
+  'ADDR306'
+]);
+
+const DECEASED_ERROR_CODES = new Set([
+  'VET360_MVI300'
+]);
+
```

Comment:
I just made all of these Sets in case new error codes with similar meaning are added in the future.